core/src/main/res/values/strings.xml

```diff
@@ -66,7 +66,7 @@
   <string name="pref_credits_title">Credits</string>
   <string name="tts_lang_not_supported">The language of this page is not supported, or appropriate language data was not installed. The article may not be properly read.</string>
   <string name="no_reader_application_installed">Could not find an installed application for this type of file</string>
-  <string name="custom_app_missing_content">Your application is corrupted.\nThis might happen when you remove files on the SD Card.\nYou need to uninstall then reinstall the App from the Play Store.</string>
+  <string name="custom_app_missing_content">Your application is corrupted.\nThis might happen when you remove files on the SD Card.\nYou need to uninstall then reinstall the App from the Play Store.</string>
```

Comment:
I am not sure why that space is different to a regular space and not too bothered checking but revert this change